src/components/v2/shared/DistributionSplitsSection/DistributionSplitModal.tsx

```diff
@@ -0,0 +1,301 @@
+import { t, Trans } from '@lingui/macro'
+import { DatePicker, Form, Modal, Select } from 'antd'
+import { useForm } from 'antd/lib/form/Form'
+import { FormItems } from 'components/shared/formItems'
+import {
+  ModalMode,
+  validateEthAddress,
+  validatePercentage,
+} from 'components/shared/formItems/formHelpers'
+import { isAddress } from 'ethers/lib/utils'
+import { Split } from 'models/v2/splits'
+import { useContext, useState } from 'react'
+import { parseWad, percentToPermyriad } from 'utils/formatNumber'
+import { getDistributionPercentFromAmount } from 'utils/v2/distributions'
+import * as constants from '@ethersproject/constants'
+import { ThemeContext } from 'contexts/themeContext'
+
+import FormattedNumberInput from 'components/shared/inputs/FormattedNumberInput'
+import InputAccessoryButton from 'components/shared/InputAccessoryButton'
+
+import { CurrencyName } from 'constants/currency'
+
+export type AddOrEditSplitFormFields = {
+  projectId: string
+  beneficiary: string
+  percent: number
+  amount: number
+  lockedUntil: moment.Moment
+}
+
+type SplitType = 'project' | 'address'
+
+export default function DistributionSplitModal({
```

Comment:
The modal that appears when you click a card ('Edit' mode) or click 'Add a split' ('Add' mode)